astropy/coordinates/builtin_frames/uvw.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+import numpy as np
+
+import astropy.units as u
+
+from astropy.coordinates.baseframe import (BaseCoordinateFrame,
+                         RepresentationMapping, frame_transform_graph)
+
+from astropy.coordinates.attributes import (CoordinateAttribute,
+                         TimeAttribute, EarthLocationAttribute)
+
+from astropy.coordinates.transformations import FunctionTransform
+from astropy.coordinates.representation import (SphericalRepresentation,
+                              UnitSphericalRepresentation,CartesianRepresentation)
+from . import (ITRS,ICRS,AltAz)
+
+class UVW(BaseCoordinateFrame):
+    """
+    A coordinate or frame in the UVW system, the conventional radio astronomy frame.
```

Comment:
Similarly, I think this needs just a sentence or paragraph or something on what the u,v, and w coordinates *are*.  While it's conventional in radio astronomy, the docs still should be comprehensible to non-radio astronomers.  

This could also go in the `Parameters` section if you think that might be clearer.

Reply:
The convention for `UVW` is well stated here https://casa.nrao.edu/Memos/CoordConvention.pdf, which is also where I got the unit tests from.
When `phase=0` (by which I understand to mean `ra=0` and `dec=0`) the z-axis of `UVW` points at the vernal equinox (minus the LST). `ENU` is equivalent to `UVW(phase = (ra=local_sidereal_time, dec=latitude))`, which is similar to `AltAz` (see above).

More intuitively, the `UVW` is the projected `ENU` frame as seen from the source (the pointing given by `phase`). The convention is the one used by radio astronomers to specify the baselines of interferometer elements. 

I'll add to the docstring.